conmon/conmon.c

```diff
@@ -1157,6 +1159,9 @@ int main(int argc, char *argv[])
 	if (opt_log_path == NULL)
 		nexit("Log file path not provided. Use --log-path");
 
+	if (opt_socket_path == NULL)
+		nexit("Socket path not provided. Use --socket-path");
```

Comment:
Should't  the "option" be optional, and we hard code the default to /var/run/crio?

Reply:
We could, but this way the default lives in exactly one place, oci.go. As it was, we had separate constants in CRI-O and Conmon, both pointing to the same place.

Reply:
Can't you define it as a constant in a conmon.h and then wrap that constant with GO, so it can be used in both.

Reply:
I'm OK with this, but do we want to add a header file just for this? @mrunalp @runcom WDYT?

Reply:
We can add a header file later as we have some other bits that are shared and we could move all of them together.